c10/util/Knobs.cpp

```diff
@@ -0,0 +1,40 @@
+#include <c10/util/Exception.h>
+#include <c10/util/Knobs.h>
+#include <cstdlib>
+#include <string>
+
+namespace c10 {
+static std::function<bool(const char*, const char*, bool)> resolver =
+    justknobs_check_default;
+bool justknobs_check(const char* the_namespace, const char* feature_name, bool default_value) {
+  return resolver(the_namespace, feature_name, default_value);
+}
+bool justknobs_check_default(const char* the_namespace, const char* feature_name, bool default_value) {
```

Comment:
The intent is to allow the distributor to choose how to do feature rollouts. fbcode -> via jk, oss -> via environments. I don't think having a mix makes sense because it can get complicated to figure out which applied after the fact.